code/modules/cargo/packs.dm

```diff
@@ -2271,6 +2324,18 @@
 	crate_name = "coffin"
 	crate_type = /obj/structure/closet/crate/coffin
 
+/datum/supply_pack/misc/medsci_headsets
+	name = "Med Sci Headset Keys"
+	desc = "Three Headset Keys for ''Genetics''. To be yelled at for not cloning somone just to be yelled at for cloning them! Also get yelled at for being in sci, well having access."
```

Comment:
Three Headset Keys for ''Genetics''. To be yelled at for not cloning someone, only to be yelled at again for cloning them! Also get yelled at for being in sci, or at least having access.